tests/sqlalchemy_factory/test_sqlalchemy_factory_v2.py

```diff
@@ -85,3 +87,95 @@ class ModelFactory(SQLAlchemyFactory[Model]):
 
     instance = ModelFactory.build()
     assert instance.overridden is not None
+
+
+@pytest.mark.parametrize(
```

Comment:
Is this applicable to SQLA 1.4? If so, I think these tests should be moved and adjusted to common so tested against both 1.4 and 2